puppet/zulip_ops/templates/teleport_db.yaml.template.erb

```diff
@@ -0,0 +1,32 @@
+# See https://goteleport.com/docs/config-reference/ and
+# https://goteleport.com/docs/database-access/guides/postgres-self-hosted/
+teleport:
+  ca_pin: "sha256:df15ba56d56227e288ce183d7eee77a6bef552aaaa5dc25f0f5ea56494ce14c6"
+  auth_servers:
+    # Use the proxy address, to support running the db_service, which requires
+    # a reverse tunnel.
+    - teleport.zulipchat.net:443
+
+ssh_service:
+  enabled: no
+app_service:
+  enabled: no
+proxy_service:
+  enabled: no
+auth_service:
+  enabled: no
+
+db_service:
+  enabled: yes
+  databases:
+    - name: "<%= @hostname %>"
+      protocol: "postgres"
+      uri: "localhost:5432"
+      ca_cert_file: /etc/ssl/certs/teleport-ca.crt
+      static_labels:
+        hostname: "<%= @hostname %>"
+      dynamic_labels:
+        - name: "is_replica"
+          command:
+            ["sudo", "-u", "zulip", "psql", "-tc", "select pg_is_in_recovery()"]
+          period: 1h
```

Comment:
I guess this means we need to manually restart this job when doing a failover?

Reply:
`period: 1h` is how long this property is cached on the teleport DB server for.  Kicking the teleport DB server will refresh it automatically, yes.  We can also take this down to 1m or so.  I *think* it is re-run automatically on this cadence, so we don't want to set it too short.

This property is _intended_ to be used for RBAC, to only permit access to read-only replicas; unfortunately, gravitational/teleport#7198 means that ... doesn't seem to work, so we'll need to make the auth hostname-based for now.